src/Docker.PowerShell/Docker.Format.ps1xml

```diff
@@ -0,0 +1,144 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Configuration>
+<ViewDefinitions>
+  <View>
+    <Name>Container</Name>
+    <ViewSelectedBy>
+      <TypeName>Docker.DotNet.Models.ContainerListResponse</TypeName>
+    </ViewSelectedBy>
+    <TableControl>
+      <TableHeaders>
+        <TableColumnHeader>
+          <Label>ID</Label>
+          <Width>20</Width>
+        </TableColumnHeader>
+        <TableColumnHeader>
+          <Label>Image</Label>
+          <Width>15</Width>
+        </TableColumnHeader>
+        <TableColumnHeader>
+          <Label>Command</Label>
+          <Width>20</Width>
+        </TableColumnHeader>
+        <TableColumnHeader>
+          <Label>Created</Label>
+          <Width>22</Width>
+        </TableColumnHeader>
+        <TableColumnHeader>
+          <Label>Status</Label>
+          <Width>20</Width>
+        </TableColumnHeader>
+        <TableColumnHeader>
+          <Label>Names</Label>
+          <Width>20</Width>
+        </TableColumnHeader>
+      </TableHeaders>
+      <TableRowEntries>
+        <TableRowEntry>
+          <TableColumnItems>
+            <TableColumnItem>
+              <PropertyName>ID</PropertyName>
+            </TableColumnItem>
+            <TableColumnItem>
+              <PropertyName>Image</PropertyName>
+            </TableColumnItem>
+            <TableColumnItem>
+              <PropertyName>Command</PropertyName>
+            </TableColumnItem>
+            <TableColumnItem>
+              <ScriptBlock>[DateTime]::Now - [TimeSpan]$_.Created</ScriptBlock>
```

Comment:
Turns out there is a missing special handler in the generated Docker.DotNet code.  I'll update this to just be a propertyname in anticipation of the update to Docker.DotNet.